lib/reporter.js

```diff
@@ -110,6 +110,17 @@ class AllureReporter extends events.EventEmitter {
             this.currentHooks.pop();
         })
 
+        this.on('runner:screenshot', (screenshot) => {
+
+            var filename = this.getScreenshotFilename(screenshot),
+                data = new Buffer(screenshot.data, 'base64')
+
+            this.print('screenshot', filename)
+            this.allure.addAttachment(filename, data)
```

Comment:
Afaik at least the extension should match the mime type. It feels like a filename of the attached asset.

Do you have a suggestion for better auto-naming? The timestamp maybe? I'm not really sure what suites the best.